src/checks_util.cc

```diff
@@ -252,4 +266,49 @@ zetasql_base::StatusOr<VisitResult> RuleVisitor::defaultVisit(
   return VisitResult::VisitChildren(node);
 }
 
+std::vector<ParseToken> GetKeywords(absl::string_view sql) {
+  ParseResumeLocation location = ParseResumeLocation::FromStringView(sql);
+  std::vector<ParseToken> parse_tokens;
+  std::vector<ParseToken> keywords;
+
+  absl::Status status =
+      GetParseTokens(ParseTokenOptions(), &location, &parse_tokens);
+
+  if (!status.ok()) return keywords;
```

Comment:
I thought if tokenizer doesn't work, there should be some errror message in some other tool or the sql is good as it is in a different dialect. In both case printing an error is not linter's job to do (preventing multiple errors). Also this function called several times(again preventing multiple errors).

Returning empty string simply makes the check return without errors, and I think it is intended behavior when tokenizer fails, or is it?

Reply:
well, we should let the user know that some lint checks were skipped for their file even though they didn't enable them. The error could instruct the user to disable this lint check if it is some valid code, but tokenizer fails.

Reply:
Then I will have a tokenizer check that will say "tokenizer failed, and [this, this and this] checks will not work." etc.

Reply:
I think this might be error-prone. What if someone adds a new check that depends on tokenizer? they will have to know that they need also find a tokenizer check and see what error message it prints. I'm sure I'd forget this myself.
I think, you could print a warning message directly in each individual check saying that the check is skipped since the tokenizer failed.
Or make it so, that when registering a new check one must specify what dependencies it has. Then there will be a registry of checks that depend on parser or tokenizer.

Reply:
"I think, you could print a warning message directly in each individual check saying that the check is skipped since the tokenizer failed."
Ok, I am adding it.